.github/actions/build-rpm/src/gpg.ts

```diff
@@ -0,0 +1,96 @@
+import * as openpgp from 'openpgp'
```

Comment:
gpg.ts contians methods to be used for GPG signing for rpm packages. This is used as part of the local action. While building the rpm package we provide the signing certificate. In order to sign that we would provide the details such as the gpc private key, key ID ,  passphrase in the github secrets.